### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by 153957** [HiSPARC/datastore#26]
Docs have been added in #25. But not everything is fully documented.

Look at the docs for todo's and things that are missing:
http://docs.hisparc.nl/datastore/

**Comment by tomkooij** [comment]
Yeah! Let's add things as @kaspervd or other stumble not yet documented stuff.
